src/app.ts

```diff
@@ -89,6 +96,31 @@ function createMailto(url: string): void {
   )
 }
 
+function addCustomCss(windowElement: BrowserWindow): void {
+  const cssBaseFile = 'style.css'
+  let cssOs: string | null = null
+
+  if (is.macos) {
+    cssOs = 'style.mac.css'
+  } else if (is.linux) {
+    cssOs = 'style.linux.css'
+  } else if (is.windows) {
+    cssOs = 'style.windows.css'
+  } else {
+    cssOs = null
+  }
+
+  windowElement.webContents.insertCSS(
+    fs.readFileSync(path.join(__dirname, '..', 'css', cssBaseFile), 'utf8')
+  )
+
+  if (cssOs !== null) {
```

Comment:
When would this be null?  We are only deploying to Windows, Mac, and Linux, so I wouldn't think this would be necessary.